test/boxHtmlTable.js

```diff
@@ -42,7 +42,10 @@ function generateBoxTable(box, excluded_fields, additional_props, no_header) {
 					if (i%4 === 3) html += ' ';
 				}
 			} else {
-				html += box[prop];
+				if (box[prop].hasOwnProperty("toString") && typeof box[prop].toString === "function")
+					html += box[prop].toString();
+				else
+					html += box[prop];
```

Comment:
This change does not seem related to AVS3. It may be better in a separate PR linking to an issue reporting a problem.

Reply:
Indeed, it is not related to AVS3 but could be used widely if you don't want to display a primative type.
Happy to create a seperate issue and process that way.